cwatch.php

```diff
@@ -422,10 +463,53 @@ public function cancelService($package, $service, $parent_package = null, $paren
     public function getClientTabs($package) 
     {
         return [
-            'tabClientActions' => Language::_('Cwatch.tab_client_actions', true)
+            'tabClientActions' => Language::_('Cwatch.tab_client_actions', true),
+            'tabClientMalWare' => Language::_('Cwatch.site.malware', true)
         ];
     }
-
+    function tabClientMalWare($package, $service, array $get = null, array $post = null, array $files = null) 
+    {
+        $this->view = new View('tab_malware', 'default');
+        $this->view->base_uri = $this->base_uri;
+        // Load the helpers required for this view
+        Loader::loadHelpers($this, array('Form', 'Html'));
+        $row = $this->getModuleRow();
+        $service_fields = $this->serviceFieldsToObject($service->fields);
+        $licensekey = $service_fields->licensekey;
+        if (isset($row->meta->username)) {
+            $user = $row->meta->username;
+        }
+        if (isset($row->meta->password)) {
+            $pass = $row->meta->password;
+        }
+        if (isset($package->meta->cwatch_sandbox)) {
+            $sandbox = $package->meta->cwatch_sandbox;
+        }
+        if (isset($package->meta->cwatch_license_type)) {
+            $product = $package->meta->cwatch_license_type;
+        }
+        if (isset($package->meta->cwatch_license_term)){
+            $term = $package->meta->cwatch_license_term;
+        }
+        Loader::loadModels($this, ['Clients']);
+        $client = $this->Clients->get($service->client_id, false);
+        $email = $client->email;
+        $this->loadApi($user, $pass, $sandbox);
+        if (!empty($post)){
+            if($post['actionname']=='checkstatus'){
+                $sites = $this->api->getScanner($post['domainname']);
+            }else{
+                $sites = $this->api->addScanner(['domain' => $post['domainname'], 'password' => $post['password'], 'username' => $post['username'], 'host' => $post['host'], 'port' => $post['port'],'path'=>$post['path']]);
```

Comment:
Lines should be a maximum of 120 characters, and you should set spaces before and after operators, like `=>`. I may need to update the ruleset for phpcs to check for these.